src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -106,7 +104,7 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
 
   @State("a recall exists")
   fun `a recall exists`() {
```

Comment:
Worth renaming the fun + migrating the `@State()` title for it, e.g. "a fully populated recall exists" - except we setting documents empty for it still?